src/main/java/graphql/execution/FetchedValue.java

```diff
@@ -15,14 +14,12 @@
 @PublicApi
 public class FetchedValue {
     private final Object fetchedValue;
-    private final Object rawFetchedValue;
     private final Object localContext;
```

Comment:
rawFetchedValue is never used and its an extra attribute with no runtime purpose

It represents the value before it was unboxed - eg Optional<X> say

For the sake of performance and memory - lets kill it